frontend/www/js/omegaup/problem/validator.js

```diff
@@ -0,0 +1,45 @@
+import Vue from 'vue';
+import problem_Validator from '../components/problem/Validator.vue';
+import { OmegaUp } from '../omegaup.js';
+
+OmegaUp.on('ready', function() {
+  const payload = JSON.parse(
+    document.getElementById('problem-payload').innerText,
+  );
+  let problemsValidator = new Vue({
```

Comment:
nit:

```suggestion
  let problemValidator = new Vue({
```

por consistencia con el nombre del componente Vue.

Reply:
listo